packages/web-ui/src/HelperText/HelperText.tsx

```diff
@@ -52,6 +58,9 @@ const StyledElement = styled('span')({
   [classSelectors.invalid]: {
     '--helper-text-color': 'var(--helper-text-color-invalid)',
   },
+  [DATA_ATTRIBUTE_SELECTORS.disableUserSelect]: {
+    userSelect: 'none',
+  },
```

Comment:
Font size should be 16px not 13 for helper text

Reply:
weird, in Linear it says I fixed this 8 weeks ago 🤔 I don't know what's happened here 🤷🏻 good catch though, thanks 👍🏻